main.nf

```diff
@@ -941,24 +946,22 @@ if (params.protocol != 'amplicon') {
 
         output:
         set val(sample), val(single_end), val(is_sra), file("*.sorted.{bam,bam.bai}") into ch_ivar_metrics,
+                                                                                           ch_ivar_variants,
+                                                                                           ch_ivar_consensus,
                                                                                            ch_ivar_varscan2,
-                                                                                           ch_ivar_consensus
+                                                                                           ch_ivar_bcftools
         file "*.{flagstat,idxstats,stats}" into ch_ivar_flagstat_mqc
         file "*.log" into ch_ivar_log
 
         script:
-        prefix = "${sample}.ivar"
+        prefix="${sample}.trim"
         """
         samtools view -b -F 4 ${bam[0]} > ${sample}.mapped.bam
         samtools index ${sample}.mapped.bam
 
         ivar trim \\
             -i ${sample}.mapped.bam \\
             -b $bed \\
-            -m 50 \\
-            -q 15 \\
-            -s 4 \\
-            -e \\
```

Comment:
The -e parameter it's important because we don't know if fragmentation has been done or not with working with amplicons. So we need to keep it, otherwise most of reads are lost. The remaining parameters I suppose can be left as default.

Reply:
@saramonzon are you able to push a new commit to this pull request to revert this change?  Or in a quick follow-up pull request?  If so, we could merge this pull request and address the samtools mpileup issue later.

Reply:
ok, merge it, and I can change it in the pull request i'm working on :)

Reply:
Will do!